sdk/data/azappconfig/internal/generated/custom_client.go

```diff
@@ -16,3 +17,7 @@ func NewAzureAppConfigurationClient(endpoint string, client *azcore.Client) *Azu
 		endpoint: endpoint,
 	}
 }
+
+func (a *AzureAppConfigurationClient) Tracer() tracing.Tracer {
+	return a.internal.Tracer()
+}
```

Comment:
I see that in AzCore we don't have this Tracer in Internal : https://github.com/Azure/azure-sdk-for-go/blob/main/sdk/azcore/core.go#L140C1-L143C2
Should we make these changes consistent?

Also for a different question why is this file in our generated?

Reply:
We don't generate client constructors for data-plane services as there's no consistent authentication across services. So, we hand-write them in a `custom_client.go` file next to the generated code.

In this instance, `internal` is the internal `*azcore.Client` used by the `azappconfig.Client`. So there's no need for an `internal` in the `azcore` version.